### PR TITLE
NTPPacket fix and NTPServer

### PR DESCRIPTION
**Pull request by micheleselea**

### CODE REVIEW

File: Net/src/NTPServer.cpp [hunk 1]
```diff
@@ -0,0 +1,145 @@
+#include "Poco/Net/NTPServer.h"
```
Comment (aleks-f) [inline Net/src/NTPServer.cpp]: naming, opening braces on new line, only english comments please, remove not used commented code

Comment (micheleselea) [reply Net/src/NTPServer.cpp]: done: for NTPServer test I'll do it next week

File: Net/src/NTPPacket.cpp [hunk 2]
```diff
@@ -16,10 +16,12 @@
 #include "Poco/Net/NetException.h"
 #include "Poco/Timestamp.h"
 #include "Poco/ByteOrder.h"
-#ifndef POCO_OS_FAMILY_WINDOWS
-#include <sys/time.h>
-#else
+#if defined(POCO_OS_ANDROID)
+#include <time.h>
+#elif defined(POCO_OS_FAMILY_UNIX)
```
Comment (aleks-f) [inline Net/src/NTPPacket.cpp]: I'm not sure how this compiled, but it definitely does not look right

Comment (micheleselea) [reply Net/src/NTPPacket.cpp]: Ok I'm going to change it

Comment (micheleselea) [reply Net/src/NTPPacket.cpp]: but, now is this way:

#if defined(POCO_OS_ANDROID)
#include <time.h>
#elif defined(POCO_OS_FAMILY_UNIX)
#include <winsock2.h>
#else
#include <sys/time.h>
#endif

do you think is not correct?

Comment (aleks-f) [reply Net/src/NTPPacket.cpp]: `winsock2.h` on unix? I do not think that's correct. But why are we including windows socket header at all here? it should already be included via Poco::Net headers

Comment (micheleselea) [reply Net/src/NTPPacket.cpp]: ah ah yes! correct :) should be POCO_OS_WINDOWS I'm going to correct...don't know how can compile :)

**Pull request closed without merging**